build/vsts_build.yaml

```diff
@@ -255,7 +255,7 @@ phases:
     displayName: "Create EndToEnd Test Package"
     inputs:
       scriptName: "$(Build.Repository.LocalPath)\\scripts\\cibuild\\CreateEndToEndTestPackage.ps1"
-      arguments: "-c $(BuildConfiguration) -tv 15 -out $(Build.Repository.LocalPath)\\artifacts\\VS15 -NuGetToolsPath $(NuGetCIToolsFolder)"
```

Comment:
I'd imagine this variable is set somewhere in the ADO variable groups?

Reply:
Yep, and I'll remove it.